blog/_posts/2016-10-21-Team-Echo-wrapping-up.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Our end of RGSoC,:( !
+layout: post
+created_at: Thu July 28 2016
+permalink: blog/Team-Echo-wrapping-up-2016
```

Comment:
the permalink needs to have the date of the post in it, as explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#permalink).